style.css

```diff
@@ -0,0 +1,80 @@
+#id-main-content {
+  box-shadow: 0 0 5px black;
+  width: 300px;
+  height: auto;
+  margin: auto;
+  background-image: url(/images/image_geometry_1.png);
+  background-size: 90px;
+  background-repeat: no-repeat;
+  background-attachment: initial;
+  display: flex;
+  flex-direction: column;
+}
+
+#id-header-content {
+  width: 98%;
+  height: auto;
+  margin: 10px auto 50px auto;
+  display: flex;
+  justify-content: flex-end;
+}
+
+:nth-child(1) #id-side-menu {
+  width: 20px;
+  height: 20px;
+  align-self: flex-end;
+}
+
+#id-main-section {
+  margin: 50px auto 0 auto;
+  width: 90%;
+  font-family: Arial, Helvetica, sans-serif;
+}
+
+#id-my-name {
+  width: 100%;
+  height: auto;
+  margin: auto;
+  font-size: 35px;
+  font-weight: bold;
+  font-family: inherit;
```

Comment:
Please use the proper font family which resembles that of the original design font.

Reply:
All changes required are applied! Thanks!